modules/migration-mtc-release-notes-1-8-5.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-5_{context}"]
+= {mtc-full} 1.8.5 release notes
+
+[id="technical-changes-1-8-5_{context}"]
+== Technical changes
+
+{mtc-first} 1.8.5 has the following technical changes:
+
+.Federal Information Processing Standard (FIPS)
+
+FIPS are a set of computer security standards developed by the United States federal government in accordance with the Federal Information Security Management Act (FISMA).
+
+Starting with version {mtc-short} 1.8.5 is designed for FIPS compliance.
+
+[id="resolved-issues-1-8-5_{context}"]
```

Comment:
```suggestion
[id="mtc-resolved-issues-1-8-5_{context}"]
```
I use "microshift" in all of my IDs since I'm also in the repo but not OCP. You might want to apply the same as content grows. Not required, but a thought for your consideration.